specification/agricultureplatform/AgriculturePlatform.DataPlane/routes.tsp

```diff
@@ -0,0 +1,512 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Microsoft.AgriculturePlatform;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoClientRequestId &
+  NoConditionalRequests;
```

Comment:
Just checking: you don't support Conditional Requests but your resources contain eTags?